tools/linter_lib/custom_check.py

```diff
@@ -511,10 +535,12 @@ def build_custom_checkers(by_lang):
          'exclude': set(['static/templates/settings/display-settings.handlebars',
                          'templates/zerver/keyboard_shortcuts.html',
                          'templates/zerver/markdown_help.html']),
-         },
+         'good_lines': ['<img src="{{source_url}}" alt="{{ _(name) }}" />', '<img alg="" />'],
+         'bad_lines': ['<img alt="Foo Image" />']},
         {'pattern': '\Walt=["\']{{ ?["\']',
          'description': "alt argument should be enclosed by _().",
-         },
+         'good_lines': ['<img src="{{source_url}}" alt="{{ _(name) }}" />'],
+         'bad_lines': ['<img alt="{{ " />']},
```

Comment:
I'm not sure if `<img alt="{{ " />'` is correct here, I think should be an empty string as the rule description suggests, `<img alt="" />`. :)

Reply:
The patter for that was really being weird when tested. That was the only thing I tested that worked

Reply:
The rule requires ``` alt=``` and then ```'``` or ```"``` followed by ```{{{``` and an optional space completed by a closing quote. I can't think of how to update/change it

Reply:
Oops no worries, this was actually my bad and I forgot to delete my comment. Sorry about that! :)

Reply:
I thought I was reading it wrong at first too